app/config.js

```diff
@@ -1,13 +1,31 @@
-export const config = {
-  botcoin_contract: "0xe150cbd19aaa7f202ab241422cfaaa99deb464e4",
-  curation_council_contract: "0x5ce5bc32909660100b0a1ec998c1eca9f59946ec",
-  developer_registry_contract: "0x5c8f6d050c843cf48a5c49ee0d126cf0c26f2204",
-  token_vault_contract: "0x2a1392c3bf2c8abce13f06a3e54850959ca001bc",
-
-  etherscan_url: "https://kovan.etherscan.io",
-  etherscan_api_url: "http://api-kovan.etherscan.io/api",
-  etherscan_api_key: "UI5FHY91GIQ94U6IUDKRGDYS4JWSGGXD24",
-  geth_rpc_url: "https://kovan.infura.io/quylRadtDHfbMF9rF15R",
-  coinbase_price_api_url: "https://api.coinbase.com/v2/prices/ETH-USD/spot",
-  ipfs_gateway_url: "https://ipfs.infura.io/ipfs/"
+import configMainnet from './config.mainnet.js';
+import configKovan from './config.kovan.js';
+
+class Config {
```

Comment:
Maybe you should move all localStorage related calls into this class?
So it be able to determine default network  in constructor

Reply:
It's not working... this class exists before the browser context is available to access localStorage, it was my initial approach.